app/controllers/school_suggestions_controller.rb

```diff
@@ -0,0 +1,11 @@
+class SchoolSuggestionsController < ApplicationController
+  def index
+    render json: GiasSchool.search_name_urn_postcode(query_params)
+  end
+
+  private
+
+  def query_params
+    params.require(:query)&.downcase
+  end
```

Comment:
With `require`, can this ever be missing? Also, since it's a single param.

```suggestion
  def query_param
    params.require(:query).downcase
  end
```

Reply:
unfortunately yes. If nothing is input, then `params.require(:query)` will just return `nil`. Which obviously breaks the `downcase`

Reply:
😢